x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/create_security_rule_type_wrapper.ts

```diff
@@ -77,6 +76,19 @@ export const createSecurityRuleTypeWrapper: CreateSecurityRuleTypeWrapper =
         injectReferences: (params, savedObjectReferences) =>
           injectReferences({ logger, params, savedObjectReferences }),
       },
+      autoRecoverAlerts: false,
+      getViewInAppRelativeUrl: ({ rule, start, end }) => {
+        if (!start || !end) {
+          return '';
+        }
+        return getNotificationResultsLink({
+          from: new Date(start).toISOString(),
+          to: new Date(end).toISOString(),
```

Comment:
I can see, that earlier this function was called with time formatted in `ms`. Are we sure ISO string is going to work here? Do you know how it can be tested?

Reply:
That might be my fault. I saw the definition of `from` and `to` was a string and assumed ISOString but you're right that it looks like a different format. `start` and `end` are passed in as epoch millis so @e40pud you should parse it differently as needed.

Reply:
done